pkg/deploy/graph/analysis/dc.go

```diff
@@ -36,7 +32,7 @@ dc:
 			if istNode := uncastIstNode.(*imagegraph.ImageStreamTagNode); !istNode.Found() {
 				dcNode := uncastDcNode.(*deploygraph.DeploymentConfigNode)
 
-				// 1. Image stream for tag of interest does not exist
+				// The image stream for the tag of interest does not exist.
 				if isNode, exists := doesImageStreamExist(g, uncastIstNode); !exists {
```

Comment:
Remember that once we have `oc create imagestream`, we can provide some help here.